web/package.json

```diff
@@ -4,14 +4,21 @@
   "license": "MIT",
   "scripts": {
     "ls:watch": "nx run labelstudio:build:development --watch",
+    "ls:unit": "nx run labelstudio:unit",
+    "ls:e2e": "nx run labelstudio-e2e:e2e",
     "lsf:watch": "nx run editor:build:development --watch",
     "lsf:serve": "nx run editor:serve:development",
-    "lsf:e2e": "cd libs/editor/tests/e2e && npm run test",
+    "lsf:e2e": "cd libs/editor/tests/e2e && yarn test",
```

Comment:
In a further iteration it would be nice to move this test suite to an app test suite like labelstudio-e2e, and have an editor-e2e. That way we could use the nx project settings and detection for `affected` and run a single command for testing which would invoke the e2e, integration, or unit layers accordingly to which project has affected files. That way CI does not have to run tests where code has not been changed.

Reply:
Yeah I agree with you, I tried to use the nx test suite but as we are using codeceptjs to test e2e I wasn't able to make it works, I would have to change a lot of things on our test suite so unfortunately I believe that for now it's better keep how it is right now!